<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAdminWindow</class>
 <widget class="QWidget" name="CreateAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/add_icon.png</normaloff>:/icons/add_icon.png</iconset>
  </property>
  <widget class="QGroupBox" name="initialsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>371</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Введіть ПІБ адміністратора</string>
   </property>
   <widget class="QLineEdit" name="lastnameLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lastnameLabel">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Прізвище</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstnameLabel">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ім'я</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstnameLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="middlenameLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="middlenameLabel">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>По батькові</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="loginGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Введіть логін та пароль</string>
   </property>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Логін</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Створити адміністратора</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
